src/main/java/com/google/devtools/build/lib/remote/options/CommonRemoteOptions.java

```diff
@@ -33,4 +38,38 @@ public class CommonRemoteOptions extends OptionsBase {
               + " the client to request certain artifacts that might be needed locally (e.g. IDE"
               + " support)")
   public List<String> remoteDownloadRegex;
+
+  @Option(
+      name = "experimental_remote_cache_ttl",
+      defaultValue = "3h",
```

Comment:
Higher value by default means higher space requirement on remote cas. I picked `3h` randomly because it's on par with the default value for `--max_idle_secs`.

The TTL only affects incremental builds. If, for example, a long-running invocation lasts more than `3h`, nothing goes wrong during this invocation. It's just for next incremental build, all remote metadata are discarded. Also, the plan is to have a background thread refresh the lease for remote metadata. So for long-running invocations, it doesn't matter what the value of `--experimental_remote_cache_ttl` is set to. The frequency of the refresh is based on  `--experimental_remote_cache_ttl`, though.

That being said, I am open to give it a higher default value.

@brentleyjones, you commented on this flag in the prototype, do you have any suggestions here?

Reply:
I don't like it being as high as 24h, because I know there are some remote cache instances that can't reach that level of guarantee. And like you said, this doesn't impact long-running builds, only incremental builds. I'm good with `3h`, but I don't think it should be larger than `8h`.